rdkit/Chem/Descriptors.py

```diff
@@ -7,28 +7,51 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-from collections import abc  # this won't work in python2, but we don't support that any more
 
+from collections import abc  # This won't work in python 2, but we don't support that any more
+from typing import Callable
+
+import rdkit.Chem.ChemUtils.DescriptorUtilities as _du
 from rdkit import Chem
+from rdkit.Chem import rdMolDescriptors
 from rdkit.Chem import rdMolDescriptors as _rdMolDescriptors
-from rdkit.Chem import rdPartialCharges, rdMolDescriptors
-import rdkit.Chem.ChemUtils.DescriptorUtilities as _du
-from rdkit.Chem.EState.EState import (MaxEStateIndex, MinEStateIndex, MaxAbsEStateIndex,
-                                      MinAbsEStateIndex)
+from rdkit.Chem import rdPartialCharges
+from rdkit.Chem.EState.EState import (MaxAbsEStateIndex, MaxEStateIndex,
+                                      MinAbsEStateIndex, MinEStateIndex)
 from rdkit.Chem.QED import qed
 
 
-def _isCallable(thing):
-  return isinstance(thing, abc.Callable) or \
-              hasattr(thing, '__call__')
-
-
+def _isCallable(something):
+  return isinstance(something, (abc.Callable, Callable)) or hasattr(something, '__call__')
+
+def _belongToRDKit(something):
+  NAME = 'rdkit'
+  
+  try:
+    if something.__module__ == NAME:
+      return True
+    if NAME in something.__module__:
+      return True
+  except Exception:
+    pass
+  
+  try:
+    if something.__class__.__module__ == NAME:
+      return True
+    if NAME in something.__class__.__module__:
+      return True
+  except Exception:
+    pass
+  
+  return False
```

Comment:
This is a significant modification. Why is this required and what is it filtering out?

Reply:
Yes, just for notice here, if some descriptors used any third-party modules or libraries, we want to ensure those should be consistent called, I have made so many mistake on this issue before getting worked done. This is to guarantee and prevent further issue that can be potentially made. 
Moreover, `Descriptors` is easily to be made as the Chemistry development, it would be noticed that this issue can be pointed out in the early version